allofplos/article_class.py

```diff
@@ -858,6 +858,27 @@ def url(self):
         """
         return self.get_url(plos_network=self.plos_network)
 
+    @property
+    def taxonomy(self):
+        """Taxonomy information
+        """
+        tag_path_elements = ('/',
+                             'article',
+                             'front',
+                             'article-meta',
+                             'article-categories')
+        e_list = self.get_element_xpath(tag_path_elements=tag_path_elements)
+        taxonomy_item = []
+        for subj in e_list[0].getchildren():
+            if subj.values()[0] == 'heading':
+                pass
+            elif subj.values()[0] == 'Discipline-v3':
+                subj_list = []
```

Comment:
I'd prefer this to be a list of tuples than a list of lists.
A tuple comprehension would probably be enough since you're only iterating over the 'subject' items.

It should be ordered, right?